package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "@nextcloud/babel-config",
+  "version": "1.0.0",
+  "description": "Babel shared config for nextcloud apps",
+  "main": "index.js",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/nextcloud/babel-config.git"
+  },
+  "peerDependencies": {
+    "@babel/core": "^7.13.10",
+    "@babel/plugin-syntax-dynamic-import": "^7.8.3",
+    "@babel/preset-env": "^7.13.12",
+    "babel-plugin-add-module-exports": "^1.0.4"
```

Comment:
I don't think we should have more than this.
Anything related to eslint like `babel-eslint` is for https://github.com/nextcloud/eslint-config

Any other very essential plugins?